### ISSUE TITLE
Heartbeat 2: Leadership training for Reps and Regional leaders

### ISSUE THREAD

**Issue by rosanardila** [mozilla/participation-org#23]
# Project description

This heartbeat aims to design and test distributed leadership trainings for Reps. This first heartbeat focuses on identifying impact and setting personal and community goals that tie back to the organizational goals in a more tangible way.
The idea is to design a training that allows volunteers to take part remotely.
# Primary Goals/Objectives - Success Measures
- Design and create a v.01 workshop: Impact and setting goals
- Validate hypothesis about disconnection to organizational goals
# Secondary Goals/Objectives
- Test workshop with a small group of volunteers
# Key hypotheses and how they will be tested

Hypothesis: it isn’t completely evident for volunteers how Mozilla has an impact. This unclarity about impact is also present in many of the volunteer led projects. This results many times in volunteers contributing to Mozilla related activities that might not have a great impact and in suboptimal local and regional coordination around goals.

Test: Analyze the local/regional community goal and personal goals of a small sample of volunteers before the workshop.

Hypothesis: by understanding a) how Mozilla has impact, b) how the organizational goals support that impact and c) how they can start setting goals that support the organizational goals and therefore have impact, volunteers will be able to focus their energy on more impactful projects.

Test: after the workshop ask volunteers to rethink/reframe their local/regional community and personal goals and analyze the differences.
# Deliverables
- Version 0.1 of the Impact and goals workshop
- Analysis of the local/regional goals
  *if possible analysis of goals after the workshop
# Team
- Owner: @rosanardila
- Members: @nukeador , @brianking 
- Informed: @williamq
# Milestones (complete before dates)
- [x] May 20th: first draft of workshop content
- [x] May 22nd: second draft after feedback from team
- [x] May 22nd: find group for first test round and schedule the training
- [x] May 25th: survey about goals
- [x] May 27th: first round of workshop
- [x] May 28th: analyze changes in goals
# References

**Comment by rosanardila** [comment]
What was supposed to happen?

We wanted to a) survey some mentors on the way make they plan their Mozilla activities, b) run the planning for impact workshop online c) see if the is a change in the way the mentors think about their goals and plan

What did happen?
We gathered a group of 10 people in vidyo and conducted the planning for impact workshop. 9 of them had responded the first survey. After the workshop 2 participants who answered the second survey. We gather some more feedback in 1:1 conversations.

The workshop went well, but it was hard to get the conversation started over vidyo. Conversations took longer to start probably due to the fact that participants were joining via vidyo and there is a lag and is difficult to have lively conversations because you're waiting your turn to speak to not interrupt anyone.

What we learned:

The way that Reps plan and chose their activities is highly different. Some wait for a prompt from Mozilla in terms of priorities and projects, some go with the best learning opportunities, some go with what their local community wants.

Very little communities (be it regional, local, functional) have careful long term plans, most of them work on an ad-hoc way. Reps aren't also always very connected to local communities, but they feel that they are there to support them with their efforts.

It seems to be that goals are mainly discussed and decided in meet-ups. People really want to see each other to be able to decide on goals, the power of meeting personally was present for all.

After the workshop a couple of participants were really happy to see this as a tool that will help them plan better and have more impact. Some of them are already thinking about how this will change events they had already planned. 

What are some improvements?

Since the conversations were hard to get started we can pilot having groups join together vidyo, so that they can have the conversations among each other a.nd then the bigger conversation with the group.

Having help material for participant to use during the first discussions
We should probably use a different conferencing solution, google hangouts seems to work better.

What should we keep the same in this process?

The survey at the beginning is interesting, giving participants their answers afterwards could have a powerful learning effect.
We had a good icebreaker where we changed vidyo rooms, that gave a lot of energy, trying those to keep the energy up.

What does the next Heartbeat for this project look like?

A version 2 of this workshop, tying the answers to the survey better to the material learned. We should aim to reach all mentors.
A nice template on the Web, where Reps can reverse engineers their planning and link it to their Reps initiatives.
